docs/examples/secrets/README.md

```diff
@@ -0,0 +1,7 @@
+# Secrets examples
+
+To run this examples you need to [create a Tenant](../README.md#tenant) and [set the environment variable](../README.md#pipelinerun) `TENANT_NAMESPACE` accordingly.
+
+## Missing secret
+
+
```

Comment:
It looks like some documentation should go here.

Reply:
Fixed